stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/messages/adapter/ListenerContainerImpl.kt

```diff
@@ -0,0 +1,137 @@
+package io.getstream.chat.android.ui.messages.adapter
+
+import com.getstream.sdk.chat.adapter.ListenerContainer
+import com.getstream.sdk.chat.view.MessageListView.AttachmentClickListener
+import com.getstream.sdk.chat.view.MessageListView.GiphySendListener
+import com.getstream.sdk.chat.view.MessageListView.MessageClickListener
+import com.getstream.sdk.chat.view.MessageListView.MessageLongClickListener
+import com.getstream.sdk.chat.view.MessageListView.MessageLongClickListenerView
+import com.getstream.sdk.chat.view.MessageListView.MessageRetryListener
+import com.getstream.sdk.chat.view.MessageListView.ReactionViewClickListener
+import com.getstream.sdk.chat.view.MessageListView.ReadStateClickListener
+import com.getstream.sdk.chat.view.MessageListView.UserClickListener
+import kotlin.properties.ReadWriteProperty
+import kotlin.reflect.KProperty
+
+public class ListenerContainerImpl(
```

Comment:
Just a copy and paste from the ListenerContainerImpl in our old UI components module